### DISCUSSION TITLE
Google Play Beta builds

### DISCUSSION THREAD

**Discussion by MarioSwitch** [osmandapp/OsmAnd#21009] [Q&A] [answered]
I don't have any **beta build** from Google Play Store since early September (version 4.9.0). 

Normally, **beta builds** are released every week (this is not a "rule", but it was nearly always the case), so I was wondering if the latest version is indeed 4.9.0 (in the **beta** program) or if my Play Store is broken?

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/17d49185-d3f1-4036-8779-59682cb7cbeb"  width="300" height="auto"/> </td>
  </tr>
</table>

_No updates since Sep 4..._

**Comment by DmitryAlexei** [comment, marked as answer]
Yes, 4.9.0 is the latest beta build
You can also try Alpha or Nightly builds (see https://osmand.net/docs/versions/nightly_versions#nightly-builds )
